src/FrontendAccountCreation.Web/Controllers/AccountCreation/AccountCreationController.cs

```diff
@@ -1253,4 +1263,29 @@ private void SetBackLink(AccountCreationSession session, string currentPagePath)
                                       // Remove when we migrate all environments to custom policy
                                       User.Claims.FirstOrDefault(claim => claim.Type == "emails")?.Value;
 
+    private static string SerializeModelState(ModelStateDictionary modelState)
+    {
+        var errorList = modelState.ToDictionary(
+            kvp => kvp.Key,
+            kvp => kvp.Value.Errors.Select(e => e.ErrorMessage).ToArray()
+        );
+
+        return JsonConvert.SerializeObject(errorList);
```

Comment:
System.Text.Json is used for serialization elsewhere in this project - it would be better to use that then there is no need to bring in the Newtonsoft nuget package.